ui/components/app/confirmations/alerts/alert-modal/alert-modal.tsx

```diff
@@ -95,58 +126,74 @@ function AlertHeader({ selectedAlert }: { selectedAlert: Alert }) {
           marginTop={3}
           marginBottom={4}
         >
-          {selectedAlert.reason ?? t('alert')}
+          {customAlertTitle ?? selectedAlert.reason ?? t('alert')}
         </Text>
       </Box>
     </>
   );
 }
 
-function AlertDetails({ selectedAlert }: { selectedAlert: Alert }) {
+function AlertDetails({
+  selectedAlert,
+  customAlertDetails,
+}: {
+  selectedAlert: Alert;
+  customAlertDetails?: React.ReactNode;
```

Comment:
[suggestion] can rename customAlertDetails → customDetails

Reply:
Renamed `customAlertDetals` to `customDetails` https://github.com/MetaMask/metamask-extension/pull/24469/commits/c68451fb395de3dc7fd05553c1471584a09b9abf .